frontend/src/components/grantaccess/GrantDelegatedAdmin.vue

```diff
@@ -183,3 +271,28 @@ const handleSubmit = async () => {
         </div>
     </VeeForm>
 </template>
+<style lang="scss">
+@use '@bcgov-nr/nr-theme/design-tokens/light-buttons.scss' as lightButton;
+@use 'sass:map';
+
+.dialog-accept-button {
```

Comment:
the dialog popup accept button now previously was set to be "red" color for the delete option in the common repo, set it to be our regular primary blue here